parser_library/src/checking/asm_instr_class.cpp

```diff
@@ -121,10 +125,10 @@ bool assembler_instruction::check_flag_operand(
 }
 
 bool assembler_instruction::check_process_flag_parameters(const std::vector<std::unique_ptr<asm_operand>>& input,
-    const std::string& instr_name,
+    std::string_view instr_name,
     const diagnostic_collector& add_diagnostic) const
 {
-    const static std::vector<std::string> other_pair_options = {
+    const static std::vector<std::string_view> other_pair_options = {
         "NOPUSH", "NORECORD", "NOPU", "NORC", "PUSH", "RECORD", "PU", "RC"
```

Comment:
Consider having e.g. each operand on a separate line to maintain the same look and feel as on lines  [90 - 109](https://github.com/eclipse/che-che4z-lsp-for-hlasm/blob/971b6e8ac9f3e9b693ca12b4c8de744841865905/parser_library/src/checking/asm_instr_class.cpp#L90-L109)